button/popup.js

```diff
@@ -241,8 +249,8 @@ $(function() {
     });
 
     $("#div_report_an_ad").click(function() {
-        var url = "pages/adreport.html?url=" + encodeURIComponent(tab.unicodeUrl)
-        + "&tabId=" + tab.id;
+        var url = "pages/adreport.html?url=" + encodeURIComponent(tab.unicodeUrl) +
+            "&tabId=" + tab.id;
```

Comment:
Line continuation indent should be clear that it's not an indent for a code block. Use either 2 spaces or make it line up with the equals sign.